### DISCUSSION TITLE
Event for showing that an update is available but is not ready yet

### DISCUSSION THREAD

**Discussion by mary-ext** [vite-pwa/vite-plugin-pwa#619] [Q&A]
is there any way I could show that there's an update going on? there's `onNeedRefresh` that gets called when the update is actually ready, but I want to show some indication that there *is* an update going on, so users don't keep refreshing or asking why the site/app hasn't been updated yet

**Comment by userquin** [comment]
no, I need to figure out how to detect it, it's a feature I've been putting off for a long time

**Reply by userquin** [reply]
created issue #620
